dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/GitHubHelperTest.java

```diff
@@ -22,7 +22,7 @@ void testGetGitHubAccessToken() {
             GitHubHelper.getGitHubAccessToken(CODE, GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET);
             fail("No CustomWebApplicationException thrown");
         } catch (CustomWebApplicationException e) {
-            assertEquals("HTTP 400 Bad Request", e.getMessage());
```

Comment:
If I had to guess what was going on, maybe we wanted to return http status codes on `getMessage` calls but just the raw message on `getErrorMessage` calls. 

Ok with the simplification in general, but I wonder if we have a mix of messages, some of which have http codes and some of which do not, maybe we should standardize on having them as a prefix or suffix rather than removing them entirely.

Reply:
We use `CustomWebApplicationException` for two purposes:
1. As a vessel for the web server response, in the case that it bubbles out of our code.
2. To signal our enclosing code, which will catch the exception and do something like log or create an object using the embedded message (then potentially rethrow).

This PR [hopefully] does not affect the behavior associated with Purpose 1, the status code and detailed message are incorporated in the `Response` which is issued by the app server, same as before.

So, regarding Purpose 2, do we ever want the detailed message that our code extracts from the exception to contain a status code?  We haven't issued (and may not issue) a response based on the exception, there isn't the need for a status code yet...